stl/inc/syncstream

```diff
@@ -218,11 +218,14 @@ protected:
         }
 
         const _Size_type _New_capacity  = _Calculate_growth(_Buf_size, _Buf_size + 1, _Max_allocation);
-        const _Elem* const _Old_ptr     = streambuf_type::pbase();
+        _Elem* const _Old_ptr           = streambuf_type::pbase();

```

Comment:
Is that needed for deallocating or a remnant from the Initial implementation?

Reply:
needed, deallocate wants pointer to non-const elements.

https://github.com/microsoft/STL/blob/17fde2cbab6e8724d81c9555237c9a623d7fb954/stl/inc/xmemory#L829-L833